virttest/utils_libvirt/libvirt_numa.py

```diff
@@ -79,3 +81,57 @@ def create_hmat_xml(vmxml, params):
     vmxml.sync()
 
     return vmxml
+
+
+def get_continuous_numa_node_index(node_list):
```

Comment:
I assume you want to get the index of first adjacent numa node.
it sounds like it is a little bit clearer if method name changed to get_first_continuous_numa_node_index or get_first_adjacent_numa_node_index

Reply:
Updated with rename and move the function to internal